tests/test_parser.py

```diff
@@ -241,18 +233,9 @@ def test_label_escaping(self):
             self.assertEqualMetrics([metric_family], list(families))
 
     def test_help_escaping(self):
-        for escaped_val, unescaped_val in [
-                ('foo', 'foo'),
-                ('\\foo', '\\foo'),
-                ('\\\\foo', '\\foo'),
-                ('foo\\', 'foo\\'),
-                ('foo\\\\', 'foo\\'),
-                ('\\n', '\n'),
-                ('\\\\n', '\\n'),
-                ('\\\\\\n', '\\\n'),
-                ('\\"', '\\"'),
-                ('\\\\"', '\\"'),
-                ('\\\\\\"', '\\\\"')]:
+        for escaped_val, unescaped_val in [('foo', 'foo'), ('\\foo', '\\foo'), ('\\\\foo', '\\foo'),
```

Comment:
This was clearer as it was

Reply:
Agree but flake8 gives following error:

`tests/test_parser.py:232:5: E129 visually indented line with same indent as next logical line`

Should I add E129 in flake8 ignore? Let me know how to proceed @brian-brazil

Reply:
Updated it.